connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestClientTest.java

```diff
@@ -157,23 +161,55 @@ private void setupHttpClient(int responseCode, String responseJsonString) throws
             when(httpClient.newRequest(anyString())).thenReturn(req);
         }
 
+        @Test
+        public void testNullUrl() throws Exception {
+            int statusCode = Response.Status.OK.getStatusCode();
+            setupHttpClient(statusCode, toJsonString(TEST_DTO));
+
+            assertThrows(NullPointerException.class, () -> httpRequest(
+                    httpClient, null, "GET", null, null, new TypeReference<Void>() { }, null, null
```

Comment:
Because the exception being thrown is somewhat generic, I'd probably pass valid values for all arguments except the one that is under test, just to make sure that the NPE is being thrown by the proper check